bin/index.js

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env node
+var DraftLog = require('draftlog').into(console).addLineListener(process.stdin);
+var chalk = require('chalk');
+var SpeedTestNet = require('../');
+
+/*
+ * Keep track of current UI State
+ */
+var header, speeds, location
+var step = 'Ping';
+var statuses = {
+  Ping: true,
+  Download: false,
+  Upload: false,
+}
+
+/*
+ * Renders the header and Speeds (one below the other)
+ */
+var halfwidth = 12;
+function updateLines() {
+  var headerTxt = '';
+  var speedsTxt = '';
+  var spinner = Spinner();
+
+  // Build Header
+  for (var k in statuses) {
```

Comment:
This variable is [declared twice](https://github.com/ddsol/speedtest.net/pull/35/files#diff-9ea21889e9fbc7d82e7dabb0fb280808R45)